Added docker standalone support. (#12)
C0urante/kafka-connect-reddit: README.md

and then copying and unzipping the zip archive generated in the `target/components/packages`
directory onto the plugin path or classpath for your Kafka Connect worker(s).

## Running standalone in docker

> To run this in standalone-mode via a docker container. (This brings up the environment + the connect-standalone):

Reviewer: Nit: wording
```suggestion
To run this in standalone-mode via a docker container (this brings up the environment and the connector in a standalone worker):
```

```bash

> #Build the project 

Reviewer: Nit: whitespace consistency
```suggestion
# Build the project 
```

mvn clean package

# Build the docker container 
docker build -t redditconnector .

# Run 
docker run --env-file reddit.env -it redditconnector

```

The environment file that was used looks like this: (To configure the subreddit, comment location, and where your broker can be reached)

```bash

> SUBREDDIT=chicagohelicopters

Reviewer: Any reason not to use `POSTS_SUBREDDITS` here (with the `POST_` prefix, and pluralized)? Seems better to be explicit than implicit, and it'd align with the precedents set elsewhere in the connector for the properties that dictate the posts/comments subreddits.

Author: That's a good point.. I didn't consider that.

COMMENTS_SUBREDDIT=chicagohelicopters

> KAFKA_SERVER=<kafka hostname>:9092

Reviewer: Nit: kind of seems like 9092 is a hard coded port here, maybe change it to:
```suggestion
KAFKA_SERVER=localhost:9092
```

```

> Note: Change the kafka host name. If the variable in the file or docker run arguements is not present then it will default to the following values:

Reviewer: Nit: spelling, wording
```suggestion
Note: Make sure to change the Kafka host name. Variables not specified in the env file or docker run arguments will default to the following values:
```

Reviewer: Also, might be worth pointing out that people can use the `docker.for.mac.localhost` hostname if they're using Mac and have a Kafka broker running locally.

- [x] Publish to [Confluent Hub]
- [ ] Support reverse-chronological consumption
- [ ] Sink connector

> - [ ] (In Docker) Replace variables in the prioperties files rather than append 

Reviewer: Nit: spelling
```suggestion
- [ ] (In Docker) Replace variables in the properties files rather than append 
```
